index.html

```diff
@@ -22,5 +22,16 @@
             <div class="line"></div>
         </div>
     </nav>
+    <div class="background">
+        <h1 class="headline">I’m Mariane Glad to see you!</h1>
+        <p class="headline-paragraph">
+            I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
```

Comment:
- [ ] Please make sure that the background image matches that one in the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design 

| Your design | Figma design | 
| ---------------- | ------------------ |
| ![image](https://user-images.githubusercontent.com/74744787/176566259-b9f484bb-d695-4918-a468-3a93b74ac30a.png) | ![image](https://user-images.githubusercontent.com/74744787/176566328-b48644e9-ba8d-4076-baa3-06709854b31b.png) |